app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CommentsController < ApplicationController
+  
+  http_basic_authenticate_with name: "user", password: "secret", only: :destroy 
+  
+  def create
+    @article = Article.find(params[:article_id])
+    @comment = @article.comments.create(comments_params)
+
+    redirect_to article_path(@article)
+  end
+
+  def destroy
+    @article = Article.find(params[:article_id])
+    @comment = @article.comments.find(params[:id])
+    @comment.destroy
+
+    redirect_to(article_path(@article))
```

Comment:
Try not to mix different coding styles:
`redirect_to article_path(@article)` line 9
`redirect_to(article_path(@article))` line 17

Reply:
Thank you iI'll keep it in mind